compiler/rustc_span/src/source_map.rs

```diff
@@ -467,6 +467,11 @@ impl SourceMap {
         self.span_to_string(sp, FileNameDisplayPreference::Remapped)
     }
 
+    /// Format the span's location information into string
+    pub fn span_to_location_info_string(&self, sp: Span) -> String {
```

Comment:
Nit: this function is used only once, it's probably better to inline and remove it.

Reply:
done